utils/dynamicLink.js

```diff
@@ -140,7 +140,7 @@ const generateCommunityLink = async (communityName) => {
 
 const generateUsernameLink = async (username) => {
     try {
-        const betterWebAppUrl = `${BETTER_WEB_APP_URL}/?username=${username}`
+        const betterWebAppUrl = `${BETTER_WEB_APP_URL}/?username=${username}+`
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Appending a `+` character directly to the username might cause issues if the username contains special characters that need to be URL encoded. It's better to use `encodeURIComponent()` function to ensure the username is properly encoded before appending it to the URL.
```diff
- const betterWebAppUrl = `${BETTER_WEB_APP_URL}/?username=${username}+`
+ const betterWebAppUrl = `${BETTER_WEB_APP_URL}/?username=${encodeURIComponent(username)}+`
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->